### PR TITLE
Internal: JS API - Fix refactor BC. [ED-2084]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: modules/web-cli/assets/js/core/commands.js [hunk 1]
```diff
@@ -19,6 +19,23 @@ export default class Commands extends CommandsBackwardsCompatibility {
...
+					'$e.commands.classes',
+					'3.7.0',
+					'$e.commands.getCommandClass(), $e.commandsInternal.getCommandClass(), $e.data.getCommandClass(), $e.routes.getCommandClass() according to the requested command infra-structure,'
+				);
+
+				return {
+					... $e.commands.commands,
+					... $e.commandsInternal.commands,
+					... $e.data.commands,
+					... $e.routes.commands,
```
Comment (matipojo) [inline modules/web-cli/assets/js/core/commands.js:35]: Why return all? Each one should return its own commands.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/web-cli/assets/js/core/commands.js (hunk 1 above)
Comment (iNewLegend) [reply modules/web-cli/assets/js/core/commands.js:35]: `$e.commands.classes` in BC included all the commands.

Comment (matipojo) [reply modules/web-cli/assets/js/core/commands.js:35]: I see, but why?

Comment (iNewLegend) [reply modules/web-cli/assets/js/core/commands.js:35]: Explained in private.

**Pull request merged**